build.js

```diff
@@ -59,9 +61,9 @@ metalsmith(__dirname)
     // Restrict the files we transform to a pattern; don't transform 2015.
     .use(branch('!2015/**/*')
     .use(branch('!assets/**/*')
-        // Also put json from /data into collections.
+        // Also put stuff from source/data into collections.
         .use(json_to_files({
```

Comment:
It is OK because I patched json_to_files to handle yaml:

https://github.com/mdn/viewsourceconf/pull/136/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R31

I wrote several new tests for json_to_files that make me confident it's working (https://github.com/hoosteeno/metalsmith-json-to-files/commit/7ac8d1ba836ae3b538e3e2792553cb797253f7d6). I've reached out to the maintainer about a merging into the upstream branch.